from src.end_to_end_student_marks_prediction.logger import logging
from src.end_to_end_student_marks_prediction.exception import CustomException
import sys



# if __name__=="__main__":
#     logging.info("The excution has started")
#     try:
#         a=1/0
#     except Exception as e:
#         logging.info("Custom Exception")
#         raise CustomException(e, sys)